### ISSUE TITLE
!group search {random group} only makes the bot reply with the user who typed the command  and !group update called !group changeCooldown at the same time.

### ISSUE THREAD

**Issue by hasanlatch007** [Yes-Theory-Fam/yesbot-ts#219]
I have already written a fix for this but wanted to still create an issue to log this better with a proper description
When a user writes a random group in the command !group search (anything that isn't a group) the bot just replies with the user's tag

I believe the cause was in the function itself and the problem was in this line
https://github.com/Yes-Theory-Fam/yesbot-ts/blob/4e9ad659de06c72537bbed207d1ddc96126cc60f/src/programs/GroupManager.ts#L377-L378

Because the .findOne didn't return any value it kept running the code but with a group value of null which doesn't print the lists of groups because of the code before it basically the whole function

I fixed this by adding certain if conditions, I think it could've done way better than how I did it but it works right?

The second issue was a missing `break;` at

https://github.com/Yes-Theory-Fam/yesbot-ts/blob/4e9ad659de06c72537bbed207d1ddc96126cc60f/src/programs/GroupManager.ts#L106-L114

Which just ran the changeCooldown case also fixed

Basically the new additions all it does is:

if a user searches a nonexistant group he gets a reply saying the group doesn't exist
If a user just types !voice search it replies with the lists of groups
If a user types !voice search (an existing group) it just does what its suppose to do 

and added the break; pretty straightforward.

**Comment by geisterfurz007** [comment]
Closed with 7d0f36c4595cb744c70eeb5d98277ec660311e96
